crates/rpc/rpc-eth-api/src/helpers/transaction.rs

```diff
@@ -62,6 +62,10 @@ pub trait EthTransactions: LoadTransaction<Provider: BlockReaderIdExt> {
     /// Singer access in default (L1) trait method implementations.
     fn signers(&self) -> &parking_lot::RwLock<Vec<Box<dyn EthSigner>>>;
 
+    /// Broadcasts raw transaction to subscribed observers. Used internally by
+    /// `send_raw_transaction` to emit transactions as they are processed.
+    fn broadcast_raw_transaction(&self, raw_tx: Bytes);
```

Comment:
I'd like to avoid this, it would mean removing the default impl for send_raw txs but this is fine because we already have it specialized for OP

Reply:
yeah. Besides that, i couldn't think of a better way to expose `broadcast_raw_transactions` than adding a trait method that's is expected to be implemented for `EthAPI` so that `send_raw_transaction` can use that to notify listeners.

Reply:
what I'd like to do instead is remove the default impl of the trait and then impl it for ethapi:

https://github.com/paradigmxyz/reth/blob/cf2a6a1ee8e8375b107627e7525a726405a5053f/crates/rpc/rpc/src/eth/helpers/transaction.rs#L12-L22

with the broadcast in place, this way we don't need to include it in the trait itself.

Reply:
Niceee. I think i get what you are saying.  i will make the change.